modules/common-fs2/src/test/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/config/ConfigFileSpec.scala

```diff
@@ -90,6 +84,24 @@ class ConfigFileSpec extends Specification with CatsIO {
               )
             )
           )
+        ),
+        io.AdaptersConfig(
+          callrail = callrailConfig,
+          cloudfrontAccessLog = cloudfrontAccessLogConfig,
+          googleAnalytics = googleAnalyticsConfig,
+          hubspot = hubspotConfig,
+          mailchimp = mailchimpConfig,
+          mailgun = mailgunConfig,
+          mandrill = mandrillConfig,
+          marketo = marketoConfig,
+          olark = olarkConfig,
+          pagerduty = pagerdutyConfig,
+          pingdom = pingdomConfig,
+          sendgrid = sendgridConfig,
+          statusgator = statusGatorConfig,
+          unbounce = unbounceConfig,
+          urbanAirship = urbanAirshipConfig,
+          vero = veroConfig
```

Comment:
You can define this once somewhere and reuse it everywhere

Reply:
Have defined `adaptersSchemas` in `SpecHelpers` to be reused here.